extensions/amp-experiment/1.0/attribute-allow-list/attribute-allow-list.js

```diff
@@ -68,6 +69,9 @@ export const attributeMutationAllowList = {
   'style': {
     '*': new DefaultStyleAllowedAttributeMutationEntry(),
   },
+  'class': {
```

Comment:
I believe there're a bunch of internal class names that are only reserved for AMP only. For example everything starts with `i-amphtml-*`. Could you please double check? Thank you

Reply:
Oh my you are completely correct, I was only thinking about security not validation 😂 Thank you!